src/webserver/WsBase.h

```diff
@@ -114,8 +118,7 @@ class WsSession : public WsBase,
         if (!ec_ && ec != boost::asio::error::operation_aborted)
         {
             ec_ = ec;
-            BOOST_LOG_TRIVIAL(info)
-                << tag() << __func__ << ": " << what << ": " << ec.message();
+            perfLog_.info() << tag() << ": " << what << ": " << ec.message();
```

Comment:
It used to be `info` in a common log but now it's info in the 'Performance` channel which we can also tweak separately or turn off completely. We could ask Sophia what she wants this to be but i think it's not too important as it will be most likely turned off on any production system outside of performance testing.